openmc/cross.py

```diff
@@ -112,6 +151,33 @@ def __init__(self, left_nuclide=None, right_nuclide=None, binary_op=None):
         if binary_op is not None:
             self.binary_op = binary_op
 
+    def __hash__(self):
+        return hash(str(self))
+
+    def __eq__(self, other):
+        return str(other) == str(self)
```

Comment:
There's an existing `__eq__` method below you should delete. And same comment as before -- move `__repr__` up